x-pack/heartbeat/monitors/browser/source/validatepackage.go

```diff
@@ -42,6 +42,10 @@ func parseVersion(version string) string {
 }
 
 func validateVersion(expected string, current string) error {
+	if strings.HasPrefix(current, "file://") {
```

Comment:
do we want to move this behind a flag? Just a thought though, what if users runs a custom locally modified synth version? 

But I am also thinking, if they are using the flag, then we are in the same boat.